docs/sdk/authentication/session-sigs/get-session-sigs.md

```diff
@@ -1,113 +1,331 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 ---
 sidebar_position: 2
 ---
 
 import FeedbackComponent from "@site/src/pages/feedback.md";
 
-# Generating SessionSigs
+# Generating Session Signatures
+
+[Session signatures](./intro.md) can be obtained using the [getSessionSigs](https://v5.api-docs.getlit.dev/classes/lit_node_client_src.LitNodeClientNodeJs.html#getSessionSigs) method provided by the Lit SDK. This method will generate a _session keypair_ for you locally, and will use the `authNeededCallback` method you specify to sign the session keypair, authorizing it to use the specified [Lit Resources and Abilities](./resources-and-abilities.md) on the Lit network.
```

Comment:
I think we need to refer to the V6 doc? also we need to think how do we update them automatically in the future. eg. we can use a url shortener/self-host service that can redirect & track all the ref links, and update them all at once.

eg.
`bit.ly/v5-doc -> https://v5-foo.getlit.dev`

later when we want to update:
`bit.ly/v5-doc -> https://v5-bar.getlit.dev`


```suggestion
[Session signatures](./intro.md) can be obtained using the [getSessionSigs](https://v6-api-doc-lit-js-sdk.vercel.app/classes/lit_node_client_src.LitNodeClientNodeJs.html#getSessionSigs) method provided by the Lit SDK. This method will generate a _session keypair_ for you locally, and will use the `authNeededCallback` method you specify to sign the session keypair, authorizing it to use the specified [Lit Resources and Abilities](./resources-and-abilities.md) on the Lit network.
```

Reply:
yea we should do something like this. good idea @Ansonhkg